services/movie_session.py

```diff
@@ -0,0 +1,47 @@
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=cinema_hall,
+        movie=movie
+    )
```

Comment:
```suggestion
    return MovieSession.objects.create(
        show_time=movie_show_time,
        cinema_hall=cinema_hall_id,
        movie=movie_id
    )
```

Reply:
I tried to write in that way, but then I have next error: Cannot assign "2": "MovieSession.cinema_hall" must be a "CinemaHall" instance. I didn't know how to fix it in another way, maybe you can help me?